web/themes/custom/cloudy/src/css/_overrides.scss

```diff
@@ -15,11 +15,6 @@ body {
   position: relative;
 }
 
-.lead {
```

Comment:
@brittanysmart, careful with removing classes from Bootstrap. There were several instances of it that were not in your PR. The use of lead was in config as well as twig templates.